packages/react-divider/Spec.md

```diff
@@ -0,0 +1,222 @@
+```
+
+Important
+
+```html
+<Divider important>This is important!</Divider>
+```
+
+Inset
+
+```html
+<Divider inset />
+```
+
+Appearance
+
+```html
+<Divider apperance="subtle" />
+<Divider apperance="brand" />
+<Divider apperance="strong" />
+```
+
+## API
+
+From [Divider.types.tsx](http://about:blank)
+
+### Slots
+
+- content (?)
+
+### Props
+
+```ts
+export interface DividerProps extends ComponentProps, React.HTMLAttributes<HTMLElement> {
+  tokens?: RecursivePartial<DividerTokens>;
+
+  /**
+   * Determines the alignment of the content within the divider.
+   * @defaultvalue 'center'
+   */
+  alignContent?: 'start' | 'end' | 'center';
+
+  appearance?: 'default' | 'subtle' | 'brand' | 'strong';
+
+  children?: any;
+
+  /* A divider can have a overriding border color */
+  color?: string;
+
+  /* A divider can be classified as important to emphasize its content */
+  important?: boolean;
+
+  inset?: boolean;
+
+  /* A divider can be horizontal (default) or vertical*/
+  vertical?: boolean;
+
+  /* Overrides for custom appearances */
+  height?: string;
+  width?: string;
+
+  fontSize?: string;
+  fontWeight?: string;
+  fontColor?: string;
+
+  margin?: string;
+  marginTop?: string;
+  marginBottom?: string;
+  marginLeft?: string;
+  marginRight?: string;
+
+  borderStyle?: string;
+  borderSize?: string | number;
+}
+```
+
+## Styling Tokens
+
+The current exposed tokens and default values are listed below:
+
+**Default**
+
+Font Color : tokens.alias.color.neutral.neutralForeground2
+
+Border Color: tokens.alias.color.neutral.neutralStroke2
+
+**Subtle**
+
+Font Color : tokens.alias.color.neutral.neutralForeground2
+
+Border Color: tokens.alias.color.neutral.neutralStroke3
+
+**Brand**
+
+Font Color : tokens.alias.color.brand.brandBackgroundStatic
+
+Border Color: tokens.alias.color.brand.brandBackgroundStatic
+
+**Brand**
+
+Font Color : tokens.alias.color.neutral.neutralForeground2
+
+Border Color: tokens.alias.color.neutral.neutralStroke1
+
+## Structure
+
+```
+<div>
+  <!-- ::before to handle the divider line independent of the divider having content or not -->
+  <div>Content</div>
+  <!-- ::after to handle the divider line that goes after the content if it has been provided. -->
+</div>
+```
+
+## Migration
+
+The divider control is a direct replacement of the current Separator control. Notable changes are listed below:
+
+Migration from V0 will require removing the content property and nesting that content as a child of the node ex.
+
+```
+<Divider content="My Content">
+<Divider>My Content</Divider>
+
+```
+
+_Note:_ The property theme from the separator control has been removed
+
+```
+<Separator theme={myTheme}>
+<Divider /> /* Handled by the theme provider */
+
+```
+
+_Note:_ The property styles from the separator control has been removed and now uses the inherent react style mechanics
+
+```
+<Separator styles={myStyles}>
+<Divider style={{ direction:ltr }}/> /* Handled by the theme provider */
+```
```

Comment:
You should move this into a separate `MIGRATION.md` file and link it from here. You should also take a look at the format taken in the ones under [react-avatar](https://github.com/microsoft/fluentui/blob/master/packages/react-avatar/MIGRATION.md) and [react-link](https://github.com/microsoft/fluentui/blob/master/packages/react-link/MIGRATION.md).

Reply:
Updated